data/lang/core/en.json

```diff
@@ -871,6 +871,10 @@
     "description": "",
     "message": "Nerve Gas"
   },
+  "NONE": {
+    "description": "To indicate the absence of something",
+    "message": "<None>"
+  },
```

Comment:
Because this has the < brackets > around it, I think this would be a better fit for the `input-core` or even `ui-core` resource - or even apply the brackets in Lua to allow this string to be reused in other contexts where the brackets are not appropriate.

Reply:
@Web-eWorks There is also `<unknown>` in that file, so I thought to put this string there too.

Reply:
That works fine by me then.